.codeclimate.yml

```diff
@@ -25,6 +25,6 @@ checks:
 plugins:
   rubocop:
     enabled: true
-    channel: rubocop-0-67
+    channel: rubocop-0-68
```

Comment:
From what I can tell codeclimate/codeclimate-rubocop doesn't have this channel yet, so this doesn't seem like it would work (I'm looking at https://github.com/codeclimate/codeclimate-rubocop/branches/all?utf8=%E2%9C%93&query=channel%2Frubocop).

We should also `bundle update rubocop` so the version in the Gemfile.lock matches this.

Reply:
I clearly know this version exists on github.

Reply:
This is not clear to me. Can you offer some support that it is true? The link I provided doesn't show that codeclimate has a rubocop-0-68 channel yet.

Reply:
Also, this is what codeclimate is showing:

<img width="1082" alt="Screen Shot 2019-05-01 at 8 27 20 AM" src="https://user-images.githubusercontent.com/13696143/57016979-0365ca80-6beb-11e9-8c2e-d73a8f0615ea.png">

Reply:
Yes. CodeClimate does not yet support RuboCop 0.68.
https://github.com/codeclimate/codeclimate/blob/e31c097b4069b84106c4c074be62e05fce96d801/config/engines.yml#L177-L201